interfaces/acados_matlab_octave/sim_generate_casadi_ext_fun.m

```diff
@@ -99,7 +99,8 @@ function sim_generate_casadi_ext_fun(model_struct, opts_struct)
 end
 
 if ispc
-    mbuild(c_files_path{:}, '-output', lib_name, 'CFLAGS="$CFLAGS"', 'LDTYPE="-shared"', ['LDEXT=', ldext]);
+    % mbuild(c_files_path{:}, '-output', lib_name, 'CFLAGS="$CFLAGS"', 'LDTYPE="-shared"', ['LDEXT=', ldext]);
```

Comment:
I am wondering, why `mbuild` was used here instead of the other command in the first place.

Reply:
I wondered as well. I just know that mbuild is part of a very expensive Matlab toolbox that not many users will have access to. It might however be included in an academic license, explaining why it was used without further considerations.

Reply:
On the other hand the `system(...)` command requires `gcc` to be installed system-wide on windows, which is surely not as common and straightforward as on a linux system...

Reply:
All in all, IMO the `mbuild(...)` command is more likely to work out-of-the box on a matlab installation on windows system.

Reply:
'gcc' comes with `mingw` right? And `mingw` is part of the installation instructions of acados